run/send_sns.py

```diff
@@ -1,16 +1,19 @@
 import argparse
 import boto3
 
+
```

Comment:
if you want to change formatting/whitespace, please do that in a separate change

Reply:
will do